srtcore/api.cpp

```diff
@@ -110,13 +98,63 @@ CUDTSocket::~CUDTSocket()
    pthread_cond_destroy(&m_AcceptCond);
 }
 
+
```

Comment:
These are functions that will be later intermediately used in the socket group code. Some of them are not part of the group code, just they may use other functions that are.